### PR TITLE
add impacted_since row to recommendation table

### PR DESCRIPTION
**Pull request by dpensi**
# Description

added `impacted_since` row to `recommendation` table

Fixes CCXDEV-7860

## Type of change

Please delete options that are not relevant.

- New feature (non-breaking change which adds functionality)

## Testing steps

-

## Checklist
* [ ] `make before_commit` passes
* [ ] updated documentation wherever necessary
* [ ] added or modified tests if necessary
* [ ] updated schemas and validators in [insights-data-schemas](https://github.com/RedHatInsights/insights-data-schemas) in case of input/output change

### CODE REVIEW

File: migration/actual_migrations_test.go [hunk 1]
```diff
@@ -758,3 +758,30 @@ func TestMigration24(t *testing.T) {
...
+		return
+	}
+
+	err := migration.SetDBVersion(db, dbDriver, 24)
+	helpers.FailOnError(t, err)
+
+	_, err = db.Exec(
+		`SELECT impacted_since FROM recommendation`,
+	)
+	assert.NotNil(t, err)
```
Comment (juandspy) [inline migration/actual_migrations_test.go:777]: use `assert.NoError` instead please

File: migration/actual_migrations_test.go [hunk 2]
```diff
@@ -758,3 +758,30 @@ func TestMigration24(t *testing.T) {
...
+
+	// migrate to 25
+	err = migration.SetDBVersion(db, dbDriver, 25)
+	helpers.FailOnError(t, err)
```
Comment (juandspy) [inline migration/actual_migrations_test.go:781]: you can use `require.NoError`

File: migration/actual_migrations_test.go (hunk 1 above)
Comment (dpensi) [reply migration/actual_migrations_test.go:777]: I actually want to assert that an error did in fact occur

File: migration/actual_migrations_test.go (hunk 2 above)
Comment (juandspy) [reply migration/actual_migrations_test.go:781]: sure, ok!

**Review by juandspy — APPROVED** [review]

**Review by tisnik — APPROVED** [review]
LGTM

**Pull request merged**